### PR TITLE
DocsActiveBlock style

### PR DESCRIPTION
**Pull request by anna-wro**
Before:
screen shot 2017-07-25 at 12 20 09

After:
screen shot 2017-07-25 at 12 20 22

### CODE REVIEW

File: src/docs/_sass/_docs-active-block.scss [hunk 1]
```diff
@@ -11,6 +11,14 @@ $docsActiveBlockTransparent: rgba(0, 0, 0, 0);
     background-image: radial-gradient(circle, $grayPrimary 1px, $docsActiveBlockTransparent 1px);
   }
 
+  &--dark .docs-active-block__component {
+    background-color: $grayPrimary;
+  }
+
+  &__label {
+    padding-left:gutter(1 / 3);
```
Comment (burakukula) [inline src/docs/_sass/_docs-active-block.scss]: @anna-wro tests are failing here.

Comment (anna-wro) [inline src/docs/_sass/_docs-active-block.scss]: fixed

**Comment by kdzwinel** [comment]
@anna-wro you added `package-lock.json` again :D I'll add it to `.gitignore` so we don't repeat that :)

**Comment by kdzwinel** [comment]
Besides that - everything looks 👌

**Comment by anna-wro** [comment]
@kdzwinel Oops, sorry! :D

**Pull request merged**